### DISCUSSION TITLE
Integration Plugin with Mattermost

### DISCUSSION THREAD

**Discussion by Lujeni** [outline/outline#2612] [Show and tell]
Hi

Thanks for your great tool ! Mostly inspired from the slack integration, I developed his little brother for Mattermost -> https://github.com/Lujeni/mattermost-plugin-outline

I still have to rework the formatting of the response a bit but it make the job.

Hopefully this will help in the adoption of the tool :cat: 

Ciao

**Comment by tommoor** [comment]
Great stuff! I'll have to get a Mattermost instance started to see how it's configured :) Can it also be used with self-hosted instances?

**Reply by Lujeni** [reply]
>  Can it also be used with self-hosted instances?

@tommoor yes of course, we use a Mattermost and Outline self hosted.  

For instance, the configuration part (token is fake :))
![2021-09-28-090000_937x545_scrot](https://user-images.githubusercontent.com/757902/135038724-0f705501-b081-409d-98d0-42338e78bcd9.png)
